store/sqlstore/user_store.go

```diff
@@ -141,6 +141,40 @@ func (us SqlUserStore) Save(user *model.User) (*model.User, *model.AppError) {
 	return user, nil
 }
 
+func (us SqlUserStore) DeactivateGuests() ([]string, *model.AppError) {
+	userIds := []string{}
```

Comment:
Nit: Would you please move this down to where we need it?

Reply:
Done